lmdeploy/pytorch/models/minicpm3.py

```diff
@@ -395,8 +395,6 @@ def get_input_embeddings(self):
 class MiniCPM3ForCausalLM(nn.Module):
```

Comment:
Lack `CudaGraphMixin`?

Reply:
You have set `support_cuda_graph=False`, I thought this model did not support cudagraph.

Reply:
It was changed to `True` afterwards.